tcks/microprofile-config/pom.xml

```diff
@@ -0,0 +1,63 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-build-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-tck-microprofile-config</artifactId>
+    <name>Quarkus - TCK - MicroProfile Config</name>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <!-- TODO move the version to the build parent -->
+        <microprofile-config-api.version>1.3</microprofile-config-api.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>tck-suite.xml</suiteXmlFile>
+                    </suiteXmlFiles>
+                    <!-- These env variables are required for org.eclipse.microprofile.config.tck.CDIPropertyNameMatchingTest -->
+                    <environmentVariables>
+                        <my_int_property>45</my_int_property>
+                        <MY_BOOLEAN_PROPERTY>true</MY_BOOLEAN_PROPERTY>
+                        <my_string_property>haha</my_string_property>
+                        <MY_STRING_PROPERTY>woohoo</MY_STRING_PROPERTY>
+                    </environmentVariables>
+                    <systemPropertyVariables>
+                        <!-- Disable quarkus optimization -->
+                        <quarkus.arc.remove-unused-beans>false</quarkus.arc.remove-unused-beans>
```

Comment:
So we don't pass the TCK with a normal setup?

Reply:
If there is any TCK test that declares a bean which would, for instance, only be used through `Instance.select().get()`, then we would fail because Arc aggressively removes unsused beans. So, even if we are passing, I think it's sane to have this in place for any future TCK that it might break. After all, TCKs are often a collection of corner cases :)

Reply:
Not ideal, but don't believe it's breaking the spirit of the TCKs to remove optimizations to have them pass

Reply:
I'm not sure I understand Ken but it's a common practice to configure the impl to pass a TCK ;-)

Reply:
Just saying that it would be great if we didn't have to tweak anything, but it's not a problem if we do